### DISCUSSION TITLE
Tall target issue

### DISCUSSION THREAD

**Discussion by WagdySamih** [gilbarbara/react-joyride#891] [Q&A]
I'm facing an issue when the target element is very tall (like can exceed 100vh).
This makes the floater overflow, and the spotlight also. (out of the overlay itself)

I tried to set the max height for the spotlight, but the floater doesn't follow its position anymore (still appears bottom to the actual target height)

Also, note the following

- I have to make it scrollable (some steps need that)
- I want the floater to appear anywhere on the screen without overflowing.

I need the ability to do one of the following:

- set max height for the target spotlight and make the floater correspond to its place
- make the tooltip appears at the top and just ignore the rest of the overflowed part
- 
or any idea about how to handle this case would be great!

![image](https://user-images.githubusercontent.com/45714046/220920514-f391eea0-df8a-410e-bf1a-c1a016ae8cd2.png)

**Comment by Alice-33** [comment]
I also encountered this problem. This will happen when target approaches the bottom. Have you solved this problem now？
